<launch>

<!-- sis_v_writer_section-->
<!--sis_vp-necst-1p85m2019_writer.conf-->

  <node pkg = "necst-1p85m2019"
        name = "sis_l1_vp-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/necst/rx_sis2sb/vp_cmd' to = '/necst/rx_sis2sb/vp1_cmd'/>
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_l1/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vp-necst-1p85m2019_writer.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_l2_bias_h2_vp_necst_1p85m2019"
        type = "relay.py">
    <remap from = '/necst/rx_sis2sb/vp_cmd' to = '/necst/rx_sis2sb/vp1_cmd'/>
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_l2/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vp-necst-1p85m2019_writer.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r1_vp-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/necst/rx_sis2sb/vp_cmd' to = '/necst/rx_sis2sb/vp1_cmd'/>
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_r1/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vp-necst-1p85m2019_writer.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r2_vp-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/necst/rx_sis2sb/vp_cmd' to = '/necst/rx_sis2sb/vp2_cmd'/>
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_r2/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vp-necst-1p85m2019_writer.conf" />

  </node>


<!--sis_vgap-necst-1p85m2019_writer.conf-->

  <node pkg = "necst-1p85m2019"
        name = "sis_vgap-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_l1/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vgap-necst-1p85m2019_writer.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_vgap-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_l1/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vgap-necst-1p85m2019_writer.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
      name = "sis_vgap-necst-1p85m2019_writer"
      type = "relay.py">
  <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_r2/vp_cmd'/>
  <param name = "config_file"
         value = "$(find necst-1p85m2019)/node_conf/sis_vgap-necst-1p85m2019_writer.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_vgap-necst-1p85m2019_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/vp_cmd' to = '/1p85m2019/sis_r2/vp_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_vgap-necst-1p85m2019_writer.conf" />

  </node>


</launch>
